create/using-rh-entitlements-pipelines.adoc

```diff
@@ -0,0 +1,68 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="using-rh-entitlements-pipelines"]
+= Using Red Hat entitlements in pipelines
+:context: using-rh-entitlements-pipelines
+
+toc::[]
+
+If you have {op-system-base-full} entitlements, you can use these entitlements to build container images in your pipelines.
+
+The Insight operator automatically manages your entitlements after you import them into this operator from Simple Common Access (SCA). This operator provides a secret named `etc-pki-entitlement` in the `openshift-config-managed` namespace.
+
+You can use Red Hat entitlements in your pipelines in one of the following two ways:
+
+* Manually copy the secret into the namespace of the pipeline. This method is least complex if you have a limited number of pipeline namespaces.
+
+* Use the Shared Resources Container Storage Interface (CSI) Driver operator to share the secret between namespaces automatically.
+
+== Prerequisites
+
+* You logged on to your {OCP} cluster using the `oc` command line tool.
+
+* You enabled the Insights Operator feature on your {OCP} cluster. If you want to use the Shared Resources CSI Driver operator to share the secret between namespaces, you must also enable the Shared Resources CSI driver. For information about enabling features, including the Insights Operator and Shared Resources CSI Driver, see link:https://docs.openshift.com/container-platform/latest/nodes/clusters/nodes-cluster-enabling-features.html[Enabling features using feature gates].
++
+
++
+[NOTE]
+====
+After you enable the Insights Operator, you must wait for some time to ensure that the cluster updates all the nodes with this operator. You can monitor the status of all nodes by entering the following command:
+
+[source,terminal]
+----
+$ oc get nodes -w
+----
+
+To verify that the Insights Operator is active, check that the `insights-operator` pod is running in the `openshift-insights` namespace by entering the following command:
+
+[source,terminal]
+----
+$ oc get pods -n openshift-insights
+----
+====
+
+* You configured the importing of your Red Hat entitlements into the Insights Operator. For information about importing the entitlements, see link:https://docs.openshift.com/container-platform/latest/support/remote_health_monitoring/insights-operator-simple-access.html[Importing simple content access entitlements with Insights Operator].
```

Comment:
Should this be an xref instead of a link? https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references

"all links to internal references must be created using an xref

Reply:
It's not internal as Pipelines is a standalone server at https://docs.openshift.com/pipelines/1.13/about/understanding-openshift-pipelines.html